scripts/module/escalator_fix.nut

```diff
@@ -0,0 +1,74 @@
+// VC-MP attempts to keep players standing on objects.
+// However, if a step of an escalator is warped for re-use, it basically takes
+// the player with it, accelerating them to high speeds.
+//
+// This script creates step models which make it impossible for the player
+// to stand on the real steps, thereby avoiding the issue.
+
+function create_escalator_fix(world, p, a, count) {
+  local m = 584;
+  local l = 0.4;
+  local x = -l * sin(a);
+  local y =  l * cos(a);
+  for(local i = 0; i < count; i++) {
+    o <- CreateObject( m, world, p + Vector(x * i, y * i, 0.0), 255);
+    o.RotateToEuler(Vector(0,0,a), 0);
+  }
+}
+
+function create_double_escalator_fix(world, p, a, count, d) {
+  create_escalator_fix(world, p, a, count);
+  create_escalator_fix(world, p + Vector(d, 0.0, 0.0), a, count);
+}
+
+function create_double_stair_up_down_fix(world, p, a, d) {
```

Comment:
Still "stair"?

Reply:
Meh.. I'll force push to master to fix this.